test/pdevs_accumulator_model_test.cpp

```diff
@@ -116,6 +116,15 @@ BOOST_AUTO_TEST_SUITE( pdevs_basic_models_suite )
             BOOST_CHECK_EQUAL(28.0f, std::get<float>(g.state));
             BOOST_CHECK_EQUAL(true, std::get<bool>(g.state));
 
+            //validate output
+            typename cadmium::make_message_bags<floating_accumulator::output_ports>::type outmb1;
+            outmb1 = g.output();
+            typename cadmium::make_message_bags<floating_accumulator::output_ports>::type outmb_expected;
```

Comment:
This was one of the problems with the branches, that I didn't create the branch from master. But I have learned!